llvm/include/llvm/TargetParser/AArch64TargetParser.h

```diff
@@ -308,6 +312,94 @@ inline constexpr ExtensionInfo Extensions[] = {
 };
 // clang-format on
 
+struct ExtensionSet {
+  // Set of extensions which are currently enabled.
+  ExtensionBitset Enabled;
+  // Set of extensions which have been enabled or disabled at any point. Used
+  // to avoid cluttering the cc1 command-line with lots of unneeded features.
+  ExtensionBitset Touched;
```

Comment:
Meaning, if extension foo starts as disabled and is never changed or updated, there wouldn't be a "+nofoo" produced?

Which seems correct, just making sure I understand.

Reply:
Yes, that's correct.